### ISSUE TITLE
Minimum post age for scraper

### ISSUE THREAD

**Issue by b3lrog** [bbepis/Hayden#15]
A feature I'd like is a configurable minimum age threshold that posts have to surpass before being archived.
Setting this to something like 6 hours would essentially remove the need to manually moderate an archive since it gives jannies enough time to delete illegal content before it's archived.

**Comment by bbepis** [comment]
So there's going to be two parts to this:
- Ignoring posts that are newer than a set configurable age
- Adding capability for the board scanner to recheck threads that may contain posts older than set configurable age (Otherwise posts will be ignored until the next time the thread gets bumped)

First part should be really easy, second part would be pretty hard. It's possible to just make the first part but it's possible for posts to slip through the gaps. Depends on how much you care

**Comment by b3lrog** [comment]
Yeah, I realised this when I took a look at implementing it myself.
Having it only apply to threads would be easy to implement and wouldn't miss any posts but has the obvious downside of still archiving replies immediately.
